provisioner.go

```diff
@@ -185,8 +191,13 @@ func (p *defaultProvisioner) provisionElectron(ctx context.Context, d Dispatcher
 
 // provisionPackage provisions a package
 func (p *defaultProvisioner) provisionPackage(ctx context.Context, d Dispatcher, paths Paths, s *ProvisionStatusPackage, m mover, name, version, pathUnzipSrc, pathDirectory string, finish func() error) (err error) {
+	_, err = os.Stat(pathDirectory)
+	if err != nil && !os.IsNotExist(err) {
```

Comment:
Everything looks fine except for this part that doesn't fit in this repo. For your project you should use `go-astilectron-bindata` instead and you should remove this logic from this PR.